packages/core/src/Stores/notification-store.js

```diff
@@ -1444,6 +1444,7 @@ export default class NotificationStore extends BaseStore {
                     route: routes.proof_of_identity,
                     text: localize('Go to my account settings'),
                 },
+                closeOnClick: () => this.markNotificationMessage({ key: 'svg_needs_poi_poa' }),
```

Comment:
a doubt, will this work?
`closeOnClick: notification_obj => this.markNotificationMessage({ key: notification_obj.key }),
`
so that we don't have to hardcode any key.

Reply:
How about setting "should_show_again: false"? does that work in this case?

Reply:
Here the issue was the pop-up re appears once we have triggered CTA on the pop up. So `should_show_again` will not help here

Reply:
Valid point @shaheer-deriv. I will push your suggestion to the PR. 🙏